### ISSUE TITLE
Create Event Page

### ISSUE THREAD

**Issue by nicoleli04** [CSE110-FA24-14/Sync-De-Stress#40]
**As a:** user

**I want:** to be able to create events

**So that:** I could add new events to the system

**Can be split:**

**Blocked by:**

**Split from:** #16 

### Acceptance Criteria
- [ ] In frontend
- [ ] matches figma fields 
- [ ] form validation (title, date -- valid date, location required)
- [ ] upload picture (low priority)
